lua/lazyvim/plugins/xtras.lua

```diff
@@ -5,6 +5,7 @@ local prios = {
   ["lazyvim.plugins.extras.ui.edgy"] = 2,
   ["lazyvim.plugins.extras.lang.typescript"] = 5,
   ["lazyvim.plugins.extras.formatting.prettier"] = 10,
+  ["lazyvim.plugins.extras.lang.markdown"] = 10,
```

Comment:
Not needed I believe? Since you removed the `opts` from `render-markdown` plugin spec? You should also test it out yourself in case I'm mistaken, but that's my assumption.